### PR TITLE
#[127978433] Add tests

### PR DESCRIPTION
**Pull request by akabiru**
#### What does this PR do?

Adds tests
#### Description of Task to be completed?

Panda should be well tested to prevent breakage.
#### How should this be manually tested?
- Clone this repo
- Run `bundle` at the root of panda
- Run `rspec -fd` to run the tests
#### What are the relevant pivotal tracker stories?

 127978433

### CODE REVIEW

File: lib/panda/record/base.rb [hunk 1]
```diff
@@ -6,45 +6,32 @@ module Record
...
-          :default
-        )
-      end
-
       def initialize(attributes = {})
         attributes.each { |column, value| send("#{column}=", value) }
       end
 
       def save
         query = if id
```
Comment (andela-akabiru) [inline lib/panda/record/base.rb]: **CodeQuality - Critical**
Add support for update

File: spec/unit/routing/mapper_spec.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+require "spec_helper"
```
Comment (akabiru) [inline spec/unit/routing/mapper_spec.rb:1]: **Code Quality - Nitpick**
Cover routing with unit specs in addition to integration specs.

File: spec/unit/routing/router_spec.rb [hunk 3]
```diff
@@ -0,0 +1,89 @@
+require "spec_helper"
```
Comment (akabiru) [inline spec/unit/routing/router_spec.rb:1]: **CodeQuality - Nitpick**
Add unit specs for router in addition to integration specs.

File: spec/unit/record/base_helper_spec.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
```
Comment (akabiru) [inline spec/unit/record/base_helper_spec.rb:1]: **Code Quality - Nitpick**
Add unit specs

File: lib/panda/record/base.rb [hunk 5]
```diff
@@ -6,45 +6,32 @@ module Record
...
-          :nullable,
-          :default
-        )
-      end
-
       def initialize(attributes = {})
         attributes.each { |column, value| send("#{column}=", value) }
       end
 
       def save
```
Comment (akabiru) [inline lib/panda/record/base.rb:13]: **Code Quality - Critical**
Reduce no of lines in class

File: lib/panda/routing/router.rb [hunk 6]
```diff
@@ -11,15 +11,13 @@ def draw(&block)
...
       end
 
-      def get(path, options = {}) route "GET", path, options end
-
-      def post(path, options = {}) route "POST", path, options end
-
-      def delete(path, options = {}) route "DELETE", path, options end
-
-      def put(path, options = {}) route "PUT", path, options end
+      %w(get post delete put patch).each do |method_name|
```
Comment (akabiru) [inline lib/panda/routing/router.rb:14]: **Code Quality - Nitpick**
Use `define_method` to DRY up code.

**Pull request merged**